src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -589,7 +589,7 @@ public struct DefaultStartConfiguration : IStartConfiguration
             /// <summary>
             /// Disables ADP for the entire process for the lifetime of the process.
             /// </summary>
-            [Obsolete("This property is no longer used and will be removed in Dynamo 3.0 - please use Dynamo.Logging.Analytics.DisableAnalytics instead.")]
+            [Obsolete("This property is no longer used and will be removed in a future version of Dynamo - please use Dynamo.Logging.Analytics.DisableAnalytics instead.")]
             public bool DisableADP { get; set; }
```

Comment:
internal?

Reply:
It is not in use anymore in latest master (or I missed it somehow).

Reply:
I will remove it.